esmvalcore/cmor/_fixes/cordex/cordex_fixes.py

```diff
@@ -0,0 +1,205 @@
+                        coord.bounds = coord.core_bounds().astype(
+                            np.float64, casting='same_kind')
+        return cubes
+
+
+class AllVars(Fix):
+    """General CORDEX grid fix."""
+
+    def _check_grid_differences(self, old_coord, new_coord):
+        """Check differences between coords."""
+        diff = np.max(np.abs(old_coord.points - new_coord.points))
+        logger.debug(
+            "Maximum difference between original %s"
+            "points and standard %s domain points "
+            "for dataset %s and driver %s is: %s.",
+            new_coord.var_name,
+            self.extra_facets['domain'],
+            self.extra_facets['dataset'],
+            self.extra_facets['driver'],
+            str(diff)
+        )
+
+        if diff > 10e-4:
+            raise RecipeError(
+                "Differences between the original grid and the "
+                f"standarised grid are above 10e-4 {new_coord.units}.",
+            )
+
+    def _fix_rotated_coords(self, cube):
+        """Fix rotated coordinates."""
+        data_domain = self.extra_facets['domain']
+        domain = cx.cordex_domain(data_domain, add_vertices=True)
+        domain_info = cx.domain_info(data_domain)
+        for dim_coord in ['rlat', 'rlon']:
+            old_coord = cube.coord(domain[dim_coord].standard_name)
+            old_coord_dims = old_coord.cube_dims(cube)
+            points = domain[dim_coord].data
+            coord_system = iris.coord_systems.RotatedGeogCS(
+                grid_north_pole_latitude=domain_info['pollat'],
+                grid_north_pole_longitude=domain_info['pollon']
+            )
+            new_coord = iris.coords.DimCoord(
+                points,
+                var_name=dim_coord,
+                standard_name=domain[dim_coord].standard_name,
+                long_name=domain[dim_coord].long_name,
+                units=Unit('degrees'),
+                coord_system=coord_system,
+            )
+            self._check_grid_differences(old_coord, new_coord)
+            new_coord.guess_bounds()
+            cube.remove_coord(old_coord)
+            cube.add_dim_coord(new_coord, old_coord_dims)
+
+    def _fix_geographical_coords(self, cube):
+        """Fix geographical coordinates."""
+        data_domain = self.extra_facets['domain']
+        domain = cx.cordex_domain(data_domain, add_vertices=True)
+        for aux_coord in ['lat', 'lon']:
+            old_coord = cube.coord(domain[aux_coord].standard_name)
+            cube.remove_coord(old_coord)
+            points = domain[aux_coord].data
+            bounds = domain[f'{aux_coord}_vertices'].data
+            new_coord = iris.coords.AuxCoord(
+                points,
+                var_name=aux_coord,
+                standard_name=domain[aux_coord].standard_name,
+                long_name=domain[aux_coord].long_name,
+                units=Unit(domain[aux_coord].units),
+                bounds=bounds
```

Comment:
```suggestion
                bounds=bounds,
```
Adding a comma here at the end will ensure that our auto formatter yapf doesn't mess it up in the future.